crypto/modes/gcm128.c

```diff
@@ -337,7 +337,7 @@ void gcm_ghash_4bit(u64 Xi[2], const u128 Htable[16], const u8 *inp,
 # endif
 
 #if     (defined(GHASH_ASM) || defined(OPENSSL_CPUID_OBJ))
-# if    !defined(I386_ONLY) && \
+# if    \
         (defined(__i386)        || defined(__i386__)    || \
```

Comment:
These two lines should be merged:
```
# if    (defined(__i386)        || defined(__i386__)    || \
```

Reply:
I did a general cleanup/tweak of the spacing and extraneous parens.